functions/controllers/processStaffAugmentationFormSubmit.js

```diff
@@ -51,6 +55,7 @@ async function handlePostRequest (req, res) {
   }
 
   console.log('STAFF_AUGMENTATION_FORM_SUBMIT', JSON.stringify(data))
+  console.log(`FORM_ORIGIN ${req.body.formOrigin}`)
```

Comment:
This `console.log` is not necessary. The previous line already has that information in the form of the tags on the email body and it's meant only as a backup in case the email fails to send.